code/game/machinery/computer/climate.dm

```diff
@@ -42,12 +42,20 @@ var/list/climatecomps = list()
 			for(var/datum/weather/W in C.forecasts)
 				dat += "[W.name] "
 			dat += "</div></div></div></center>"
-		else
-			dat += "<b>Unknown Climate:</b> <div class='line'>Not configured for climate.</div></div></div></center>"
+		else // Generic weather display for other climate types
```

Comment:
would be nicer to make it all generic; if weather wants to have special lines added to climate display that could be contained within a variable on the weather itself